### DISCUSSION TITLE
Refresh_tokenのHTTP status client error (400 Bad Request)問題

### DISCUSSION THREAD

**Discussion by kuboshi112** [proshunsuke/colmsg#64] [Q&A] [answered]
おはようございます

日本語ができなくて翻訳機を使っているので話し方が変かもしれません。

mitmproxyでrefresh_tokenを読み込んだ後、colmsg--h_refresh_tokenにトークンを指定すると、HTTP status clienterror (400 Bad Request)エラーが発生します。

refresh_tokenを再発行してもらって試しても同じ結果が発生します。
現在使用中の機器はiPhone12 mini とm1 maxマックブックプロです。

似たようなエラーが私にだけ発生するのか気になります。
ありがとうございます。

**Comment by proshunsuke** [comment, marked as answer]
It' s okay in English.
This time I'll reply in Japanese.

`-g hinatazaka` を付けてもだめでしょうか？

**Reply by kuboshi112** [reply]
Thanks and sorry for late reply.

Yes, I am still having same issues with token.

**Reply by proshunsuke** [reply]
The 400 error is caused by a wrong request, which should be caused by none other than refresh_token.
In other words, refresh_token is not correct.
Please make sure that the refresh_token is correct again. And, refresh_token is updated by linking external accounts on the device. Please make sure this is not happening.

**Reply by kuboshi112** [reply]
Sorry for late reply. I had fixed an issue, as I found out that there was an error in firewall setting between ISP server and my home's internal server. Cheers.
